### PR TITLE
[ADF-721] Fix translation reference for dev task

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**
```
[ X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
[ X] Tests for the changes have been added (for bug fixes / features)
[ X] Docs have been added / updated (for bug fixes / features)
```

**What kind of change does this PR introduce?** (check one with "x")
```
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation
[ ] Other... Please describe:
```

**What is the current behaviour?** (You can also link to an open issue here)
In the dev environment, the translation doesn't work.If you try to add a new translation key in the JSON the key is not translated when the app is running.

**What is the new behaviour?**
I moved all the translation files in the bundles folder and I change all the relative pointing

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: ng2-components/config/assets/tslint.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
         "no-unused-variable": true,
         "no-use-before-declare": true,
         "no-var-keyword": true,
-        "no-var-requires": false,
+        "no-var-requires": true,
```
Comment (DenisVuyka) [inline ng2-components/config/assets/tslint.json]: this raises lots of errors for TypeScript code where we use require to resolve images, also the vendor/polyfill files

File: ng2-components/config/webpack.common.js [hunk 2]
```diff
@@ -50,18 +70,22 @@ module.exports = {
...
                 test: /\.ts$/,
                 use: ['ts-loader?' + JSON.stringify({
+                    "logInfoToStdOut": true,
+                    "logLevel": "error",
+                    "configFileName": path.resolve(__dirname, '../tsconfig.json'),
                     "compilerOptions": {
                         "typeRoots": [
-                            "../node_modules/@types"
-                        ]}
+                            "./node_modules/@types"
```
Comment (DenisVuyka) [inline ng2-components/config/webpack.common.js]: it should be taking `node_modules/@types` by default, I think this setting can be removed.

File: ng2-components/config/assets/tslint.json (hunk 1 above)
Comment (eromano) [inline ng2-components/config/assets/tslint.json]: done

File: ng2-components/config/webpack.common.js (hunk 2 above)
Comment (eromano) [inline ng2-components/config/webpack.common.js]: done

**Pull request merged**